tools/db_bench_tool.cc

```diff
@@ -5053,17 +5053,27 @@ void VerifyDBFromDB(std::string& truth_db_name) {
     // The number of iterations is the larger of read_ or write_
     Duration duration(FLAGS_duration, readwrites_);
     while (!duration.Done(1)) {
-      DB* db = SelectDB(thread);
-      GenerateKeyFromInt(thread->rand.Next() % merge_keys_, merge_keys_, &key);
+      DBWithColumnFamilies* db_with_cfh = SelectDBWithCfh(thread);
+      int64_t key_rand = GetRandomKey(&thread->rand);
+      GenerateKeyFromInt(key_rand, merge_keys_, &key);
```

Comment:
Shouldn't this be `GenerateKeyFromInt(key_rand % merge_keys_, merge_keys_, &key);` ?

Reply:
I will fix it. yes, GenerateKeyFromInt(key_rand % merge_keys_, merge_keys_, &key) is the correct one